plugins/heroku/src/setConfigVars.ts

```diff
@@ -22,7 +23,11 @@ async function setConfigVars(
       configVars.SYSTEM_CODE = systemCode
     }
 
-    await heroku.patch(`/apps/${appIdName}/config-vars`, { body: configVars })
+    const endpoint = appIdName === 'review-app' ? 
+      `/pipelines/${pipelineId}/stage/review/config-vars`:
+      `/apps/${appIdName}/config-vars`
```

Comment:
**suggestion**: i think it would be cleaner to split this into two separate methods. that would make the logging below clearer as well.

Reply:
thanks - I've updated with what I think you mean! 😄